src/components/my-share-card/MyShareCard.tsx

```diff
@@ -28,6 +28,21 @@ function MyShareCard({
   if (!data) return null
   const { type: poolType } = POOLS_MAP[data.name]
   const formattedDecimals = poolType === PoolTypes.USD ? 2 : 4
+  const hasAVAX = 
+    data.masterchefBalance?.pendingTokens.bonusTokenAddress.toLowerCase() 
```

Comment:
Do we need to have optional chain operator here? If needed, we might need to have optional chains as following.
data.masterchefBalance?.pendingTokens?.bonusTokenAddress?.toLowerCase()

Reply:
is this still needs to be handled?

Reply:
fix at #66